source/includes/_build.md

```diff
@@ -55,52 +78,43 @@ Execute `run-iroha-dev.sh` again to attach to existing container.
 
 ## Linux or macOS
 
-For Iroha, a provisioning of Postgres and Redis are required. You may launch them on your local machine, or use docker containers, as provided.
-
 > Launching Docker and Postgres in Docker
 
 ``` shell
-docker run --name some-redis -p 6379:6379 -d redis:3.2.8
-docker run --name some-postgres -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 -d postgres:9.5
+docker run --name some-redis \
+-p 6379:6379 -d redis:3.2.8
+docker run --name some-postgres \
+-e POSTGRES_USER=iroha \
+-e POSTGRES_PASSWORD=helloworld \
+-p 5432:5432 -d postgres:9.5
 ```
 
-> Linux
+To launch Iroha daemon, running postgres and redis services are required. You may launch them on your local machine, or use docker containers, as provided on the right side.
```

Comment:
What do you mean as a "right side"?

Reply:
When this .md file is finished it will look in hyperledger.github.io/iroha-api as two sided page: left one is for text and right one for the code

Reply:
Let me show:
![screen shot 2017-12-16 at 16 46 34](https://user-images.githubusercontent.com/11841667/34071049-18641bc0-e281-11e7-826e-2078dec27f87.png)